rqt_gui/setup.py

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env python
 
 from distutils.core import setup
+
 from catkin_pkg.python_setup import generate_distutils_setup
 
+package_name = "rqt_gui"
+
 d = generate_distutils_setup(
     packages=['rqt_gui'],
     package_dir={'': 'src'},
-    scripts=['bin/rqt']
+    scripts=['bin/rqt'],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
```

Comment:
I'm following the lead of demo_nodes_py 
https://github.com/ros2/demos/blob/master/demo_nodes_py/setup.py

Since this is an ament_cmake package I don't think setup.py is run. But this copies the package.xml to the package's share directory in install.